script.js

```diff
@@ -0,0 +1,47 @@
+import { firstArr } from "./data/4x4.js";
+import { secondArr } from "./data/32x32.js";
+function firstPrint(){
+  let ctx = document.getElementById("canvas").getContext("2d");
```

Comment:
basically you can extract canvas element and its 2d context once before first print and use it both in the first and second print calls